pkg/kubectl/cmd/apply.go

```diff
@@ -345,7 +345,11 @@ func RunApply(f cmdutil.Factory, cmd *cobra.Command, out, errOut io.Writer, opti
 
 			if string(patchBytes) == "{}" {
 				count++
-				cmdutil.PrintSuccess(shortOutput, out, info.Object, false, "unchanged")
+
+				// Do not print unchanged message if an output format other than "name" is specified
```

Comment:
This comment should be rephrased to explain when the "unchanged" message will be printed, not when it won't be printed.

Reply:
sure